datafusion/optimizer/src/simplify_expressions/simplify_exprs.rs

```diff
@@ -62,8 +62,7 @@ impl OptimizerRule for SimplifyExpressions {
         true
     }
 
-    /// if supports_owned returns true, the Optimizer calls
-    /// [`Self::rewrite`] instead of [`Self::try_optimize`]
+    /// if supports_owned returns true, the Optimizer calls [`Self::rewrite`]
```

Comment:
I don't understand what this comments means anymore -- I think we can simply delete it